src/govuk/helpers/_links.scss

```diff
@@ -87,7 +87,7 @@
   }
 
   &:visited {
-    color: $govuk-link-visited-colour;
+    color: $govuk-error-colour;
```

Comment:
Worth merging these together, do you think?

```scss
&:link,
&:visited {
  color: $govuk-error-colour;
}
```

Reply:
I was following what we do for the no-visited mixin, but happy to combine - makes sense to me 👍 https://github.com/alphagov/govuk-frontend/blob/master/src/govuk/helpers/_links.scss#L265

Reply:
Ooh, I hadn't noticed that. Happy either way!